deployment/upgradeLive

```diff
@@ -44,24 +68,40 @@ function replace_upgrade_script() {
 }
 
 function install() {
+	local __build=$1
+	local __install_live_recorder=$2
+    mkdir -p latest
     cd $VERSION
     nvm install
     # move configuration files (liveController's and liveRecorder's) to latest version
     cp /opt/kaltura/liveController/configMapping.json /opt/kaltura/liveController/$VERSION/common/config/
     cp /opt/kaltura/liveController/configMapping.ini /opt/kaltura/liveController/$VERSION/liveRecorder/Config/
     npm install
     cd ..
-    # download binaries from Jenkins
-    echo "### starting to download binaries"
-    /opt/kaltura/liveController/$VERSION/deployment/get_bins "$VERSION"
-    echo "### finished downloading binaries"
+    if [ "${__build}" = "yes" ]; then
+        ${PRODUCT_PATH}/build_scripts/build_ffmpg.sh ${FFMPEG_BUILD_PATH} ${FFMPEG_VERSION}
+        ${PRODUCT_PATH}/build_scripts/build_addons.sh ${PRODUCT_PATH} ${FFMPEG_BUILD_PATH}
+        ${PRODUCT_PATH}/build_ts2mp4_convertor.sh ${PRODUCT_PATH}/liveRecorder ${FFMPEG_BUILD_PATH}
+    else
+	    # download binaries from Jenkins
+	    echo "### starting to download binaries"
+	    /opt/kaltura/liveController/$VERSION/deployment/get_bins "$VERSION"
+	    echo "### finished downloading binaries"
+	fi
     # create symlinks to latest dir and process script
     if [ -r "latest" ] && [ -L "latest" ]; then
          echo "unlink latest"
          unlink latest
     fi
     ln -s /opt/kaltura/liveController/$VERSION /opt/kaltura/liveController/latest
-
+    if [ "__install_live_recorder" = "yes" ]; then
+         . ${PRODUCT_PATH}/liveRecorder/installPython2.7.sh
+	     local __modules=(schedule m3u8 poster psutil pycrypto)
+         for module in "${__modules[@]}"; do
+             python -c "import ${module}"
+             [ $? -ne 0 ] && pip2.7 install ${module}
```

Comment:
merged changes from your branch. Regarding python, using any version relies on backward compatibility. Are you sure it is always kept?
Anyway did the change.